Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Registration.hs

```diff
@@ -187,7 +188,7 @@ auth isDirectAuth req mbBundleVersion mbClientVersion = do
   person <-
     Person.findByRoleAndMobileNumberAndMerchantId SP.USER countryCode mobileNumberHash merchant.id
       >>= maybe (createPerson req mbBundleVersion mbClientVersion merchant.id) return
-  checkSlidingWindowLimit (authHitsCountKey person)
+  unless isDirectAuth $ checkSlidingWindowLimit (authHitsCountKey person)
```

Comment:
We dont want rate limiter for paytm ?

Reply:
Nope, signature authentication will rate limit, people can go back and again open sdk